test/unit/assorted/server_selection.spec.test.ts

```diff
@@ -0,0 +1,37 @@
+import { join } from 'path';
+
+import {
+  collectServerSelectionLogicTests,
+  runServerSelectionLogicTest
+} from './server_selection_logic_spec_utils';
+
+describe('Server Selection Logic (spec)', function () {
+  beforeEach(function () {
+    if (this.currentTest.title.match(/Possible/)) {
+      (this.currentTest as any).skipReason = 'Nodejs driver does not support PossiblePrimary';
```

Comment:
I'd rather the red underline remain rather than casting this. When we do start type checking our tests correctly we'll catch issues if this feature was accidentally used incorrectly.

Reply:
Done!